generator/counter.go

```diff
@@ -15,6 +15,9 @@ type Counter struct {
 // Possibly it can randomize values around increment.
 // When deviation is set, it the next value won't be less then previous.
 func NewCounter(name string, start, stop, step uint, randomizeStart bool, value, deviation float64) (*Counter, error) {
+	if value < 0 && math.Abs(value) < math.Abs(deviation) {
+		return &Counter{}, ErrNewCounter
```

Comment:
nil will be better, it won't create an object

Reply:
```suggestion
		return nil, ErrNewCounter
```

Reply:
Ok